packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -75,6 +88,25 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
   /// recognizer looks for, like 'tap' or 'horizontal drag'.
   String get debugDescription;
 
+  /// {@template flutter.gestures.recognizer.dragStartBehavior}
+  /// Configure the behavior of offsets sent to [onStart].
+  ///
+  /// If set to [DragStartBehavior.start], the [onStart] callback will be called at the time and
+  /// position when the gesture detector wins the arena. If [DragStartBehavior.down ],
+  /// [onStart] will be called when at the time and position when a down event
+  /// was first detected.
+  ///
+  /// For example:
+  /// A finger presses down at on the screen with offset (500.0, 500.0),
+  /// and then moves to position (510.0, 500.0) before winning the arena.
+  /// With [dragStartBehavior] set to [DragStartBehavior.down], the [onStart]
+  /// callback will be called at the time corresponding to the touch's position
+  /// at (500.0, 500.0). If it is instead set to [DragStartBehavior.start],
+  /// onStart will be called at the time corresponding to the touch's position
+  /// at (510.0, 500.0).
+  /// {@endtemplate}
```

Comment:
(in both paragraphs here) it doesn't actually change the time, does it? the onStart is always sent when it wins. It's just that the position of the start, and the subsequent deltas, will be relative to the position at the win time vs at the down time.